services/core-api/app/api/services/document_manager_service.py

```diff
@@ -10,7 +10,7 @@
 ALLOWED_DOCUMENT_CATEGORIES = [
     'tailings', 'permits', 'variances', 'incidents', 'reports', 'mine_party_appts', 'noticeofwork',
     'bonds', 'reclamation_invoices', 'explosives_permits', 'project_summaries',
-    'notices_of_depature'
+    'notices_of_depature', 'irt'
```

Comment:
Do we want to use the full name here rather than the abbreviation? It would be consistent with the rest. Would that be much work?